src/messaging/ExchangeMgr.cpp

```diff
@@ -372,8 +372,10 @@ void ExchangeManager::SendStandaloneAckIfNeeded(const PacketHeader & packetHeade
                                                 const SessionHandle & session, MessageFlags msgFlags,
                                                 System::PacketBufferHandle && msgBuf)
 {
-    // If we need to send a StandaloneAck, create a EphemeralExchange for the purpose to send the StandaloneAck
-    if (!payloadHeader.NeedsAck())
+
+    // If using the MRP protocol and we need to send a StandaloneAck, create a EphemeralExchange for the purpose to send
```

Comment:
```suggestion
    // If using the MRP protocol and we need to send a StandaloneAck, create a EphemeralExchange to send
```

Reply:
Noticed another typo in the original comment as well. `create an EphemeralExchange..`. Going to add that change as well.